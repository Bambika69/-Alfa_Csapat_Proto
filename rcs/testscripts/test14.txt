createfield
createfield F1
createvirologist F1
startturn V1
move F2