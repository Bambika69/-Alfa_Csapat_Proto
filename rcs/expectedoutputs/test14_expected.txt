Field created:
ID: F1
Type: Field
Neighbours: null

Field created:
ID: F2
Type: Field
Neighbours: F1

Virologist created:
ID: V1
Field: F1
Backpack: B1

V1â€™s turn started.

Virologist moved: 
Virologist: V1
Origin: F1
Destination: F2
